models/users.go

```diff
@@ -449,3 +463,38 @@ func (p PasswordReset) BuildCognitoRequest(clientSecret string, clientId string)
 		Username:   &p.Email,
 	}
 }
+
+// BuildListUserGroupsRequest build the require input for cognito query to obtain the groups for given user
+func (p UserParams) BuildListUserGroupsRequest(userPoolId string, nextToken string) *cognitoidentityprovider.AdminListGroupsForUserInput {
+	if nextToken != "" {
+		return &cognitoidentityprovider.AdminListGroupsForUserInput{
+			UserPoolId: &userPoolId,
+			Username:   &p.ID,
+			NextToken:  &nextToken,
+		}
+	} else {
```

Comment:
`else` not required - if the `if` condition is not true you can fall through to the other `return` statement. i.e.

```
if nextToken != "" {
    return &cognitoidentityprovider.AdminListGroupsForUserInput{
        UserPoolId: &userPoolId,
	Username:   &p.ID,
	NextToken:  &nextToken,
    }
}

return &cognitoidentityprovider.AdminListGroupsForUserInput{
    UserPoolId: &userPoolId,
    Username:   &p.ID,
}
```